pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -232,7 +232,7 @@ func (pp *proportionPlugin) updateShare(attr *queueAttr) {
 	res := float64(0)
 
 	// TODO(k82cn): how to handle fragment issues?
-	for _, rn := range api.ResourceNames() {
+	for _, rn := range attr.allocated.ResourceNames() {
```

Comment:
`attr.allocated`'s extended resource is initialed to nil, so its `ResourceNames()`  will not contain extended resource?

Reply:
update to use attr.deserved